clients/fides-js/src/components/Overlay.tsx

```diff
@@ -100,7 +109,15 @@ const Overlay: FunctionComponent<Props> = ({
     return () => clearTimeout(delayBanner);
   }, [setBannerIsOpen]);
 
+  const showModal = () => {
+    if (!modalLinkRef.current) {
+      document.body.classList.add("fides-overlay-modal-link-shown");
+    }
+    handleOpenModal();
+  };
+
   useEffect(() => {
+    window.Fides.showModal = showModal;
```

Comment:
```suggestion
    window.Fides.showModal = showModal;
    document.body.classList.add("fides-overlay-modal-link-shown")
```

Basically that CSS class should be synced to whether or not the modal link _should_ be shown and usable.